### ISSUE TITLE
When I got log, there are s0, s1, s2 loss mask !

### ISSUE THREAD

**Issue by BaeSukkyoung** [open-mmlab/mmdetection#4239]
Hello, now I'm using Cascade Mask R-CNN !

When I trained this model and got the log, there are s0, s1, s2 loss mask.
But, in the configuration file, there is only one loss_weight of mask.
The weight of bbox and cls are totally 8, half is bbox and the other is cls.
So, in the configuration file, there are four loss_weight of bbox & cls.

I'm really curious about the structure of loss function in this model, which is Cascade Mask R-CNN !

**Comment by v-qjqs** [comment]
Hi @BaeSukkyoung, actually there are three mask heads, see [here](https://github.com/open-mmlab/mmdetection/blob/63772c5cb80b0124b6722f705759ed0e49bb679c/mmdet/models/roi_heads/cascade_roi_head.py#L72).
And the loss weights for the three mask heads are decided by [`stage_loss_weights`](https://github.com/open-mmlab/mmdetection/blob/master/configs/_base_/models/cascade_mask_rcnn_r50_fpn.py#L38).
